sites/default/menu_data.json

```diff
@@ -322,6 +322,14 @@
         "children" : [ ],
         "requirement" : 0,
         "acl_req" : [ "admin", "users" ]
+    }, {
+        "label" : "Users Activity",
+        "menu_id" : "adm0",
+        "target" : "adm",
+        "url" : "/interface/usergroup/users_activity.php",
```

Comment:
Yes, I removed it because it was a part of my initial commit (earlier solution in which I showed last activity of each user on that page). Not needed in this solution.

Reply:
Gotcha.  Did a test, and the only thing we are now worried about is the autologging.  Otherwise, it is quite nice.